scripting/get5.sp

```diff
@@ -1167,11 +1174,13 @@ public Action Event_RoundEnd(Event event, const char[] name, bool dontBroadcast)
     }
 
     g_KnifeWinnerTeam = CSTeamToMatchTeam(winningCSTeam);
-    Get5_MessageToAll("%t", "WaitingForEnemySwapInfoMessage",
-                      g_FormattedTeamNames[g_KnifeWinnerTeam]);
+    Get5_MessageToAll("%t", "WaitingForEnemySwapInfoMessage",g_FormattedTeamNames[g_KnifeWinnerTeam]);
+    Get5_MessageToTeam(g_KnifeWinnerTeam, "%t", "VoteMessage");
+    g_bVoteStart = true;
+    g_bSideVoteTimer = CreateTimer(15.0, Timer_VoteSide);
 
-    if (g_TeamTimeToKnifeDecisionCvar.FloatValue > 0)
-      CreateTimer(g_TeamTimeToKnifeDecisionCvar.FloatValue, Timer_ForceKnifeDecision);
+    // if (g_TeamTimeToKnifeDecisionCvar.FloatValue > 0)
```

Comment:
See above

Reply:
That value is just in case we decide to add the legacy !stay and !switch back.